README.md

```diff
@@ -62,7 +73,7 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-Your answer...
+module.exports - helps you export a module in which includes a functionality(all the functionality of a mongoose module) and definition of the module that you specified while defining your module and give you the power of CRUD over those defined modules
 ```
```

Comment:
`module.exports` is not Mongoose-specific. In server-side Javascript, we use `module.exports` export some/all of the items in a Javascript file so that we can reference it in another file using `require`.